src/service/PeopleServices.cpp

```diff
@@ -110,34 +106,46 @@ bool PeopleServices::initPassword() {
                 cout << "password you entered is not the same, please try again ; you have " << count << "chance left ." << endl;
                 count--;
             } else {
-                //TODO: need to replace by database
-                cout << "Hi," << Storage::getCurrentUser()->getName() << "your account is locked , please contact your tutor or teacher for unlock" << endl;
-                lockUser(Storage::getCurrentUser()->getUserId());
+                cout << "Hi," << People->getName() << "your account is locked , please contact your tutor or teacher for unlock" << endl;
+                lockUser(People->getUserId());
                 break;
             }
         }
     }
 
-    //TODO: need to replace by database
-    Storage::getCurrentUser()->setPassword(password);
+    People->setPassword(password);
```

Comment:
need to call PeopleDao function  updatePeoplePassword()

Reply:
Solved